airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/SecretsRepositoryWriterTest.java

```diff
@@ -212,4 +216,92 @@ private static DestinationConnection injectCoordinateIntoDestination(final Strin
     return Jsons.clone(DESTINATION_WITH_FULL_CONFIG).withConfiguration(injectCoordinate(coordinate));
   }
 
+  @Test
+  public void testWriteStagingConfiguration() throws JsonValidationException, ConfigNotFoundException, IOException {
+    final UUID destinationDefinitionId = UUID.randomUUID();
+    final JsonNode secretPayload = Jsons.jsonNode(sortMap(Map.of("name", "John", "age", "30", "car", "tesla")));
+    assertEquals("{\"age\":\"30\",\"car\":\"tesla\",\"name\":\"John\"}", secretPayload.toString());
```

Comment:
you can use multi-line strings (I believe even on a single line) to make this more readable